Libraries/Components/WebView/WebView.ios.js

```diff
@@ -231,9 +231,16 @@ var WebView = React.createClass({
      * **NOTE** : "In order for video to play inline, not only does this
      * property need to be set to true, but the video element in the HTML
      * document must also include the webkit-playsinline attribute."
-     * @platform ios
```

Comment:
Don't remove this line. `allowsInlineMediaPlayback` is an iOS specific property.